Sources/WhoopDIKit/WhoopDI.swift

```diff
@@ -1,6 +1,6 @@
 import Foundation
 public final class WhoopDI: DependencyRegister {
-    nonisolated(unsafe) private static let appContainer = Container()
+    nonisolated(unsafe) static let appContainer = Container()
```

Comment:
why is this not private anymore?

Reply:
I think I reverted whatever I was doing this needed this. Moving back to private.